src/allotropy/parsers/mabtech_apex/mabtech_apex_structure.py

```diff
@@ -50,19 +60,34 @@ def _create_measurement(plate_data: SeriesData) -> Measurement:
         sample_identifier=f"{well_plate}_{location_id}",
         detection_type=constants.DETECTION_TYPE,
         device_type=constants.DEVICE_TYPE,
-        exposure_duration_setting=plate_data.get(float, "Exposure"),
-        illumination_setting=plate_data.get(float, "Preset Intensity"),
+        exposure_duration_setting=plate_data.get(float, exposure_duration_setting_key),
+        illumination_setting=plate_data.get(float, illumination_setting_key),
         processed_data=ProcessedData(
             identifier=random_uuid_str(),
             features=[
-                ImageFeature(
-                    identifier=random_uuid_str(),
-                    feature=feature,
-                    result=plate_data.get(float, feature, NaN),
-                )
+                _build_feature(feature, plate_data, led_filter)
                 for feature in constants.IMAGE_FEATURES
             ],
         ),
+        custom_information_document=CustomInformationDocument(
+            led_filter=plate_data.get(str, "LED Filter") or NOT_APPLICABLE
+        ),
+    )
+
+
+def _build_feature(
+    feature: str, plate_data: SeriesData, led_filter: str | None
+) -> ImageFeature:
+    if led_filter:
+        led_number = re.search(r"\d+", led_filter)
+        if led_number:
+            feature = feature.format(filter=led_filter, led_number=led_number.group())
+    else:
+        feature = feature.format(filter="", led_number="").replace("  ", " ")
+    return ImageFeature(
+        identifier=random_uuid_str(),
+        feature=feature,
+        result=plate_data.get(float, feature, NaN),
```

Comment:
I know this is existing logic, but it seems like we have added more values that are missing for many of the test cases. It seems like we should instead skip creating an ImageFeature if the value is not found?